bindings/ldk_node.udl

```diff
@@ -0,0 +1,135 @@
+namespace ldk_node {
+};
+
+dictionary Config {
+	string storage_dir_path;
+	string esplora_server_url;
+	Network network;
+	string? listening_address;
+	u32 default_cltv_expiry_delta;
+};
+
+interface Builder {
+	constructor();
+	[Name=from_config]
+	constructor(Config config);
+	Node build();
+};
+
+interface Node {
+	[Throws=NodeError]
+	void start();
+	[Throws=NodeError]
+	void stop();
+	[Throws=NodeError]
+	void sync_wallets();
+	Event next_event();
+	void event_handled();
+	PublicKey node_id();
+	[Throws=NodeError]
+	Address new_funding_address();
+	[Throws=NodeError]
+	u64 spendable_onchain_balance_sats();
+	[Throws=NodeError]
+	u64 total_onchain_balance_sats();
+	[Throws=NodeError]
+	void connect_open_channel([ByRef]string node_pubkey_and_address, u64 channel_amount_sats, boolean announce_channel);
```

Comment:
will we be able to connect to a peer without opening a channel? having that will be useful for [use cases like these](https://github.com/BitcoinAndLightningLayerSpecs/lsp/pull/22)